File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/maliciousPatientDto.ts
export class MaliciousPatientDto {
    id: number = 0;
    fullname: string = '';
    username: string = '';
    isblocked: string = '';

    public constructor(id: any, fullname:any, username: any, isblocked:any) {
        this.id = id;
        this.fullname = fullname;
        this.username = username;
        this.isblocked = isblocked;
    }
}

File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/DTOs/MaliciousPatientDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HospitalLibrary.Core.DTOs
{
    public class MaliciousPatientDto
    {
        public int Id { get; set; }
        public String FullName { get; set; }
        public String Username { get; set; }
        public String IsBlocked { get; set; }

        public MaliciousPatientDto() { }

        public MaliciousPatientDto(int id, string fullname, string username, String isBlocked)
        {
            Id = id;
            FullName = fullname;
            Username = username;
            IsBlocked = isBlocked;
        }
    }
}
